wazo_load_api/setup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+# Copyright 2023 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+NAME = 'wlapi'
+setup(
+    name=NAME,
+    version='1.0',
+    author='Wazo Authors',
+    description='Wazo Load API',
+    author_email='dev@wazo.io',
+    url='https://wazo-platform.org',
+    packages=find_packages(),
+    package_data={'plugins': ['*/api.yml']},
+    entry_points={
+        'console_scripts': [
+            f'{NAME}=main:main',
+        ],
+        'plugins': [
```

Comment:
The entry point should be something more unique since any python executable could use the entry point.
I suggest `wlapi-plugins`